zk/watcher.go

```diff
@@ -159,6 +170,7 @@ func (w *Watcher) runHooks() error {
 	return nil
 }
 
+// QUESTION: Is this needed? I can't find anywhere wn.disconnected is read.
```

Comment:
It's returned in `WatchChildren`, and `Peers.sync` later tries to read from it.

Reply:
Yes, you are right.